hack/run-in-python-container.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs $@ in a python container
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+cd "${REPO_ROOT}"
+
+PY_IMAGE='python:3.9-slim-buster'
+
+docker run \
```

Comment:
I think maybe we should just always ensure and activate the venv in this script, it would de-dupe a lot of logic and ensure we don't accidentally try to do things outside of the venv?

Reply:
do you mean `docker run <IMAGE> "source activate.sh && $@"` here?

Reply:
that and or/ there's also some logic elsewhere to ensure the venv exists etc., is there a reason to be running in this environment without the venv?

Reply:
There is no reason to run this without venv, so I think we are safe to do it here.

Reply:
PR is updated